lgc/include/lgc/patch/PatchBufferOp.h

```diff
@@ -118,6 +118,8 @@ class BufferOpLowering {
   llvm::Value *replaceLoadStore(llvm::Instruction &inst);
   llvm::Instruction *makeLoop(llvm::Value *const loopStart, llvm::Value *const loopEnd, llvm::Value *const loopStride,
                               llvm::Instruction *const insertPos);
+  Value *createGlobalPointerAccess(llvm::Value *bufferDesc, llvm::Value *offset, llvm::Type *type,
+                                   llvm::Instruction &inst, const llvm::function_ref<Value *(Value *)> &callback);
```

Comment:
Pass `callback` by value instead of by reference, since `llvm::function_ref` is itself already a "reference-like" type.

Reply:
I see. Updated.